### PR TITLE
[qa] Replace with openwisp-utils-qa-checks

### PR DESCRIPTION
**Pull request by waleko**
Replace old qa checks with `openwisp-utils-qa-checks` (related to openwisp/openwisp-utils#63 for this repo)

### CODE REVIEW

**Comment by waleko** [comment]
Travis is failing because of a migration not being descriptive ([travis](https://travis-ci.org/openwisp/openwisp-users/jobs/631868625#L302))

**Comment by atb00ker** [comment]
> Travis is failing because of a migration not being descriptive (travis)

Please use option `--migrations-to-ignore`.

**Comment by waleko** [comment]
I fixed most of the stuff, but `checkmakemigrations` still fails on python 2.7. Previously, there was just an easy workaround:
https://github.com/openwisp/openwisp-users/blob/e484aecf2963ad29ef72f1f387fcecf4a046e201/.travis.yml#L42 
Perhaps, we can make an `if` to skip `checkmakemigrations` if python version is `<3`, but maybe it's a bit of a redundancy.

**Comment by atb00ker** [comment]
@waleko 

I see. I think we can run the qa check script only when it's `Python 3`, it's only a matter of time before we drop `Python 2.7` anyway. :smile:

File: .travis.yml [hunk 1]
```diff
@@ -39,16 +36,11 @@ install:
...
-  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then ./tests/manage.py makemigrations openwisp_users --dry-run | grep "No changes detected"; fi
-  - coverage run --source=openwisp_users runtests.py
   - |
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-      # gets commit message of last commit before pull request merge
-      COMMIT_MESSAGE=$(git log $TRAVIS_PULL_REQUEST_SHA --format=%B -n 1)
-      printf "Checking commit message:\n\n"
-      printf "$COMMIT_MESSAGE\n\n"
-      checkcommit --message "$COMMIT_MESSAGE"
+    if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then
```
Comment (atb00ker) [inline .travis.yml]: I think it'll look better if you move it to multiple lines like:
```
openwisp-utils-qa-checks  --migration-module openwisp_users \
                          --migrations-to-ignore 2 
                          --migration-path "./openwisp_users/migrations ./tests/testapp/migrations"
```

File: .travis.yml [hunk 2]
```diff
@@ -26,9 +26,6 @@ branches:
 
 before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - flake8
-  - ./runcheckmigration
```
Comment (atb00ker) [inline .travis.yml]: Please delete these files as well! :smile:

**Pull request merged**